examples/05_aws_batch/workflow.py

```diff
@@ -43,12 +43,16 @@ def task_on_batch(x: int) -> list:
     # task to invoke a task to run back on the local machine, however, recall that
     # calling a task initially returns a lazy expression. The lazy expression is the
     # return value from the batch task, and the local scheduler will do a follow up evaluation
-    # to run `task_on_default` locally.
+    # to run `task_on_default` locally.  We can also invoke tasks that don't specify an executor,
+    # such as script().  Task executor options can be supplied manually to override the executor, as
+    # well as CPU, memory and other machine constraints.
```

Comment:
In terms of the tutorial, I was trying to present batch executors as a separate idea from scripting and only combine the two later in `06_bioinfo_batch`.

It seems you think it would be help to show batch-based scripts `script()` earlier, such in this example? If so, perhaps the first example we show is the more common case of the outer task being local. This is a good example too, but its a more complex example of how the features interact. I can take a stab at how to incorporate `script()` into this example with some more gradual increases of complexity. What do you think?

Reply:
I agree, the sequencing of introducing new concepts is confusing/too early, although I wasn't sure it made sense to defer it to the 06_bioinfo_batch example (I appreciate the time you put into introducing concepts sequentially/independently, and then composing them.  It's just that I was fairly surprised by the semantics of tasks-in-tasks like this...

Let me remove this commit from the pull request, so it's not distracting from the other two commits.